add-translations

From: Bryan Larsen <bryan@larsen.st>

Translations appear in the `config/locales` directory.  We'll edit
`app.en.yml` with our translations.

SHOW_PATCH
---
 config/locales/app.en.yml |    6 ++++++
 1 files changed, 6 insertions(+), 0 deletions(-)

diff --git a/config/locales/app.en.yml b/config/locales/app.en.yml
index 26256ec..869012f 100644
--- a/config/locales/app.en.yml
+++ b/config/locales/app.en.yml
@@ -4,6 +4,12 @@
 en:
   hello: "Hello world"
 
+  front_title: "Projects you created"
+
+  project:
+    collection:
+      empty_message: "You do not have any projects"
+
 #  attributes:
 #    created_at: Created at
 #    updated_at: Updated at
